tahoe.mk

```diff
@@ -32,7 +32,9 @@ tahoe.restart:  ## Restarts both of LMS and Studio python processes while keepin
 amc.provision:  ## Initializes the AMC
 	docker exec -it tahoe.devstack.amc python manage.py migrate
 	make COMMAND='python manage.py lms create_oauth2_client http://localhost:13000     http://localhost:13000/oauth2/access_token/ confidential --client_name AMC --client_id 6f2b93d5c02560c3f93f     --client_secret 2c6c9ac52dd19d7255dd569fb7eedbe0ebdab2db --trusted --settings=devstack_docker' SERVICE='lms' tahoe.exec.single
+	make COMMAND='python manage.py lms create_devstack_site red --settings=devstack_docker' SERVICE='lms' tahoe.exec.single
```

Comment:
@OmarIthawi This command fails on a clean instance trying to fetch `settings.FEATURES['AMC_APP_URL']`.

Reply:
@ahmedaljazzar we need to fix that on the `configuration` repository.

Reply:
Thanks @ahmedaljazzar for catching this error. I've just fixed it with https://github.com/appsembler/edx-platform/pull/649.